autoinstall-schema.json

```diff
@@ -260,6 +260,14 @@
                 }
             ]
         },
+        "ubuntu-advantage": {
+            "type": "object",
+            "properties": {
+                "token": {
+                    "type": "string"
```

Comment:
I'm considering if we want to do crude token checks at the schema level.
For example, 30 chars, upper/lower/digits only.
If the token changes format we'd need to update but I'm OK with that.
What do you think?

Reply:
Now that the format is defined, I was thinking at using:
```json
{
    "type": "string",
    "minLength": 24,
    "maxLength": 30,
    "pattern": "^C[1-9A-HJ-NP-Za-km-z]{23,29}$
}
```

but maybe it's overkill.

Reply:
I don't think that check is overkill, except maybe the {23,29} is redundant with min/maxLength. Let's also give this a brief comment explaining the format at a basic level - the constant at the front and a pointer to https://pkg.go.dev/github.com/btcsuite/btcutil/base58#CheckEncode would be good

Reply:
Done, thanks!
Added the pattern without the redundant check.
Also added the brief explanation in the "description" field of the schema.